src/contracts/Qearn.h

```diff
@@ -269,7 +269,8 @@ struct QEARN : public ContractBase
         output.totalLockedAmount = 0;
         output.averageAPY = 0;
         locals.cnt = 0;
-        for(locals._t = qpi.epoch(); locals._t >= qpi.epoch() - 52U; locals._t--)
+
+        for(locals._t = qpi.epoch() - 1; locals._t >= qpi.epoch() - 52; locals._t--)
```

Comment:
qpi.epoch() returns an unsigned number, please make sure to use unsigned constants (1U and 52U) to prevent compiler warnings.